### ISSUE TITLE
improve profile tabs item count load time

### ISSUE THREAD

**Issue by daiagi** [kodadot/nft-gallery#7437]
goes after 
* #7318

              @exezbcz  is right, the query time is rather slow

the query as it is takes about 3 seconds to return 
![image](https://github.com/kodadot/nft-gallery/assets/22791238/bbf03325-90c5-40ba-9297-41ea62026f8f)

however, you only need this query:
```graphql
query profileStatsById($id: String!, $interactionIn: [Interaction!], $denyList: [String!]) {

  obtained: nftEntitiesConnection(
    where: {currentOwner_eq: $id, burned_eq: false, metadata_not_eq: "", issuer_not_in: $denyList}
    orderBy: name_ASC
  ) {
    totalCount
  }
  created: nftEntitiesConnection(
    where: {issuer_eq: $id, burned_eq: false, metadata_not_eq: ""}
    orderBy: blockNumber_DESC
  ) {
    totalCount
  }
  events: eventsConnection(
    where: {AND: [{caller_eq: $id, OR: {currentOwner_eq: $id}}], interaction_in: $interactionIn}
    orderBy: blockNumber_DESC
  ) {
    totalCount
  }
  collections: collectionEntitiesConnection(
    where: {nfts_some: {burned_eq: false}, issuer_eq: $id, issuer_not_in: $denyList, metadata_isNull: false}
    orderBy: blockNumber_DESC
  ) {
    totalCount
  }
}

```

and this one takes about 1.2 seconds

![image](https://github.com/kodadot/nft-gallery/assets/22791238/4dd22eb2-9784-4856-9ddb-4045c8b8f7e3)

so, instead of adding to the existing query, maybe create a new query?
so that both the existing and the new could be called in parallel, hopefully reducing load time

can be a follow-up issue

_Originally posted by @daiagi in https://github.com/kodadot/nft-gallery/issues/7318#issuecomment-1742044843_

**Comment by hassnian** [comment]
👋

**Comment by kodabot** [comment]
ASSIGNED - @hassnian 🔒 LOCKED -> Wednesday, October 4th 2023, 22:03:27 UTC -> 36 hours
